app/src/main/java/org/eu/droid_ng/wellbeing/lib/WellbeingService.kt

```diff
@@ -267,6 +299,10 @@ class WellbeingService(private val context: Context) {
 		return State(value)
 	}
 
+	fun isBedtimeModeEnabled(): Boolean {
```

Comment:
keep using State instead

Reply:
using State is slower, but I'll revert this since it's your app so it's your choice